src/main/webapp/i18n/en/guidedTour.json

```diff
@@ -15,6 +15,11 @@
             "headline": "Tutorial cancelled",
             "content": "You have cancelled the tutorial. If you want to continue this tutorial at any other time, then click on <tt>Continue tutorial</tt> from the account menu."
         },
+        "completed": {
+            "headline": "Congratulations 🎉",
+            "content": "You have successfully completed the tutorial. You can restart the tutorial with <tt>Restart tutorial</tt> or at any other time from the account menu dropdown",
```

Comment:
```suggestion
            "content": "You have successfully completed the tutorial. You can restart the tutorial with <tt>Restart tutorial</tt> at any other time from the account menu dropdown",
```

Reply:
@maxr96  The sentence may be misunderstanding. I will change.